bin/prepare-frontend.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+cp CHANGELOG.md frontend/markdown/
```

Comment:
Do we need to remove the symlink before copying or does this override?

Reply:
cp, by default, overwrites.